spec.bs

```diff
@@ -0,0 +1,520 @@
+<pre class='metadata'>
+Title: Topics API
+Status: w3c/CG-DRAFT
+ED: https://github.com/patcg-individual-drafts/topics
+Shortname: topics
+Level: 1
+URL: https://github.com/patcg-individual-drafts/topics
+Editor: Yao Xiao, Google, yaoxia@chromium.org
+Editor: Josh Karlin, Google, jkarlin@chromium.org
+Abstract: This specification describes a method that could enable ad-targeting based on the people’s general browsing interest without exposing the exact browsing history.
+!Participate: <a href="https://github.com/patcg-individual-drafts/topics">GitHub patcg-individual-drafts/topics</a> (<a href="https://github.com/patcg-individual-drafts/topics/issues/new">new issue</a>, <a href="https://github.com/patcg-individual-drafts/topics/issues?state=open">open issues</a>)
+Group: patcg
+Repository: patcg-individual-drafts/topics
+Markup Shorthands: markdown yes
+</pre>
+
+<pre class=link-defaults>
+spec:html; type:attribute; text:document
+spec:webidl; type:dfn; text:resolve
+spec:dom; type:dfn; text:origin
+spec:fetch; type:dfn; for:/; text:request
+spec:dom; type:attribute; text:URL
+spec:infra; type:dfn; text:user agent
+spec:fetch; type:dfn; text:HTTP-network-or-cache fetch
+spec:url; type:dfn; for:url; text:host
+spec:url; type:dfn; for:url; text:origin
+spec:url; type:dfn; text:domain
+spec:fetch; type:dfn; for:/; text:header list
+spec:fetch; type:dfn; for:Response; text:response
+spec:infra; type:dfn; text:list
+</pre>
+
+<pre class="anchors">
+spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
+    type: dfn
+        text: node navigable; url: document-sequences.html#node-navigable
+        text: relevant settings object; url: webappapis.html#relevant-settings-object
+        text: top-level traversable; for:navigable; url: document-sequences.html#nav-top
+        text: active document; for:navigable; url: document-sequences.html#nav-document
+        text: navigable; for: Window; url: nav-history-apis.html#window-navigable
+spec: html; urlPrefix: https://w3c.github.io/hr-time/
+    type: dfn
+        text: current relative timestamp; url: #dfn-current-relative-timestamp
+        text: monotonic clock; url: #dfn-monotonic-clock
+        text: unsafe current time; for:monotonic clock; url: #dfn-unsafe-current-time-1
+        text: duration; url: #dfn-duration
```

Comment:
You don't need to list these here; they're exported from HR-Time. Run `bikeshed update` if you haven't run that since they were added.

Reply:
https://github.com/patcg-individual-drafts/topics/commit/86945ea52f91cd6a4ca350a15df86e67f67fe3c5